### PR TITLE
Fix: drag n drop like trello in example

### PR DESCRIPTION
**Pull request by anup-a**
# Description

This change will allow drag n drop like Trello. I did a deep dive into Trello's code, and tried to mimic the feature in same exact manner.

Fixes https://github.com/wasp-lang/wasp/issues/338

## Type of change

Please select the option(s) that is more relevant.

- [ ] Code cleanup
- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### CODE REVIEW

**Comment by anup-a** [comment]
This is sample demo. 

[trello.webm](https://user-images.githubusercontent.com/29516633/198030098-6958ba32-a8dc-4d9e-a60d-688190511109.webm)

I see no recent activity on the issue, so I gave it a shot. Let me know if this looks good.

File: examples/waspello/ext/MainPage.js [hunk 1]
```diff
@@ -169,12 +169,16 @@ const Lists = ({ lists, listIdToCardsMap }) => {
 
 const List = ({ list, index, cards }) => {
   const [isPopoverOpen, setIsPopoverOpen] = useState(false)
+  const [showHeaderTarget, setShowHeaderTarget] = useState(true);
```
Comment (matijaSos) [inline examples/waspello/ext/MainPage.js:172]:
```suggestion
  const [isHeaderTargetShown, setIsHeaderTargetShown] = useState(true);
```
We usually start boolean values with `is`, so tried to rewrite it to follow this rule.

**Review by matijaSos — APPROVED** [review]
This looks great, awesome work! Added a few minor style suggestions. The last thing here is then to implement tilting of a list while it is being dragged, but we can open a separate issue for that.

File: examples/waspello/ext/MainPage.js [hunk 2]
```diff
@@ -187,6 +191,11 @@ const List = ({ list, index, cards }) => {
     setIsPopoverOpen(false)
   }
 
+  const handleHeadingClick = (e) => {
```
[the remark this answers is not in the document]
Comment (anup-a) [reply examples/waspello/ext/MainPage.js]: Done

File: examples/waspello/ext/MainPage.js (hunk 1 above)
Comment (anup-a) [reply examples/waspello/ext/MainPage.js]: Done

**Comment by anup-a** [comment]
I have updated the code as per comments

**Comment by matijaSos** [comment]
> I have updated the code as per comments

Awesome, will merge today!

**Comment by matijaSos** [comment]
@anup-a merged! Thanks again for your help, really cool that you fixed this :)

**Pull request closed without merging**